### PR TITLE
Release notes for 1.27.1

### PR DESCRIPTION
**Pull request by ikhoon**

### CODE REVIEW

**Comment by ikhoon** [comment]
image

File: site/src/pages/release-notes/1.27.1.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+date: 2024-02-07
+---
+
+## 🛠️ Bug fixes
+
+- Restored dependency breaking changes in the Brave integration module. #5438
+  - `armeria-brave` is now consistently use Brave 5.
```
Comment (jrhee17) [inline site/src/pages/release-notes/1.27.1.mdx]:
```suggestion
  - `armeria-brave` now consistently uses Brave 5.
```

or

```suggestion
  - `armeria-brave` is now consistently using Brave 5.
```

but I prefer the former

File: site/src/pages/release-notes/1.27.1.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  - `armeria-brave5` and `armeria-brave6` modules have been introduced to let users choose the specific Brave 
+    dependencies.
```
Comment (jrhee17) [inline site/src/pages/release-notes/1.27.1.mdx:10]:
```suggestion
    dependency.
```

**Review by jrhee17 — APPROVED** [review]
👍 👍 👍

**Review by minwoox — APPROVED** [review]
Thank!

**Comment by ikhoon** [comment]
Thanks for the quick review! 🙇‍♂️

**Pull request merged**